artifactory/commands/transferfiles/transfer.go

```diff
@@ -29,8 +29,8 @@ const (
 	uploadChunkSize = 16
 	// Size of the channel where the transfer's go routines write the transfer errors
 	fileWritersChannelSize       = 500000
-	retries                      = 10000
-	retriesWaitMilliSecs         = 5000
+	retries                      = 1800
+	retriesWaitMilliSecs         = 2000
```

Comment:
Retries happen when a system is experiencing significant stress. Using too low backoff time would make it worse.
I suggest keeping the 5 seconds and adjust the number of retries accordingly.